src/oc/web/actions/activity.cljs

```diff
@@ -955,3 +984,7 @@
                                                :dismiss true
                                                :expire 3
                                                :id (if success :mark-unread-success :mark-unread-error)})))))
+
+(defn change-sort-type [type]
+  (cook/set-cookie! (router/last-sort-cookie (router/current-org-slug)) (name type) (* 60 60 24 6))
```

Comment:
is that 6 days? Why so short?

Reply:
@bago2k4

Reply:
Good catch, i copied the code we have been using for all the other cookies: last used org (if you have multiple, last used board, last board etc..
We can definitely use something longer here. Any suggestion?